drivers/gpio/gpio_nrfe.c

```diff
@@ -12,6 +12,9 @@
 
 #include <zephyr/device.h>
 
+#if defined(CONFIG_GPIO_NRFE_ICMSG_BACKEND) && defined(CONFIG_GPIO_NRFE_MBOX_BACKEND)
```

Comment:
Those are under a single choice, so it should be impossible to have both selected.